backend/src/main/java/capstoneexpensetracker/backend/travelergroup/TravelerGroupController.java

```diff
@@ -17,8 +19,18 @@ List<TravelerGroup> displayTravelerGroupList() {
         return travelerGroupService.displayTravelerGroupList();
     }
 
+    @ResponseStatus(code = HttpStatus.CREATED)
     @PostMapping
     TravelerGroup addTravelerGroup(@RequestBody NewTravelerGroup newTravelerGroup) {
         return travelerGroupService.addTravelerGroup(newTravelerGroup);
     }
+
+    @ResponseStatus(code = HttpStatus.NO_CONTENT)
+    @DeleteMapping("{id}")
+    public void deleteTraveler(@PathVariable String id) {
+        if (!travelerGroupService.findTravelerGroupExistById(id)) {
```

Comment:
same as travelerController, maybe you can omit this method and move this check to the service